openedx/features/edly/utils.py

```diff
@@ -38,6 +38,9 @@ def user_has_edly_organization_access(request):
         bool: Returns True if User has Edly Organization Access Otherwise False.
     """
 
+    if request.user.is_superuser or request.user.is_staff:
```

Comment:
We already have such check in `EdlyOrganizationAccessMiddleware` then why we add it here?

Reply:
We use `user_has_edly_organization_access` in our permissions for API which we need to bypass in case of server to server communication.
```
class CanAccessEdlyPanel(BasePermission):

    def has_permission(self, request, view):
        is_edly_panel_user = request.user.groups.filter(
            name__in=[settings.EDLY_PANEL_USERS_GROUP, settings.EDLY_PANEL_ADMIN_USERS_GROUP]
        ).exists()
        has_edly_panel_access = user_has_edly_organization_access(request) and (request.user.is_superuser or is_edly_panel_user)
        return has_edly_panel_access
```

Reply:
@iam-mhaseeb can you please remove the `request.user.is_superuser` from `CanAccessEdlyPanel ` and also from `EdlyOrganizationAccessMiddleware`. As it didn't make sense to check the same permission in 2 different places.

Reply:
@burhandodhy We would need both, I'll clean this line in any upcoming PR as this in a different repository.
```
has_edly_panel_access = user_has_edly_organization_access(request) and (request.user.is_superuser or is_edly_panel_user)
```